internal/command/statics.go

```diff
@@ -1,12 +1,31 @@
 package command
 
 import (
+	"bytes"
 	"context"
 	"io"
+	"strings"
 
 	"github.com/caos/zitadel/internal/domain"
+	"github.com/go-oss/image/imageutil"
 )
 
+func (c *Commands) RemoveExif(file io.Reader, size int64, contentType string) (io.Reader, int64, error) {
+	if !strings.HasSuffix(contentType, "png") &&
+		!strings.HasSuffix(contentType, "jpg") &&
+		!strings.HasSuffix(contentType, "jpeg") &&
+		!strings.HasSuffix(contentType, "tiff") {
```

Comment:
just style issue... but would it make sense to use a regex or separate function?